src/EnergyPlus/OutputProcessor.cc

```diff
@@ -8743,34 +8816,20 @@ void ProduceRDDMDD()
         }
     }
 
-    std::ofstream rdd_stream;
-    std::ofstream mdd_stream;
+    state.files.rdd.ensure_open("ProduceRDDMDD", state.files.outputControl.rdd);
+    state.files.mdd.ensure_open("ProduceRDDMDD", state.files.outputControl.mdd);
     if (ProduceReportVDD == ReportVDD_Yes) {
-        rdd_stream.open(DataStringGlobals::outputRddFileName); // Text mode so we use \n as terminator
-        if (!rdd_stream) {
-            ShowFatalError("ProduceRDDMDD: Could not open file \"" + DataStringGlobals::outputRddFileName + "\" for output (write).");
-        }
-        rdd_stream << "Program Version," << VerString << ',' << IDDVerString << '\n';
-        rdd_stream << "Var Type (reported time step),Var Report Type,Variable Name [Units]" << '\n';
-        mdd_stream.open(DataStringGlobals::outputMddFileName);
-        if (!mdd_stream) {
-            ShowFatalError("ProduceRDDMDD: Could not open file \"" + DataStringGlobals::outputMddFileName + "\" for output (write).");
-        }
-        mdd_stream << "Program Version," << VerString << ',' << IDDVerString << '\n';
-        mdd_stream << "Var Type (reported time step),Var Report Type,Variable Name [Units]" << '\n';
+        print(state.files.rdd, "Program Version,{},{}{}", VerString, IDDVerString, DataStringGlobals::NL);
+        print(state.files.rdd, "Var Type (reported time step),Var Report Type,Variable Name [Units]{}", DataStringGlobals::NL);
+
+        print(state.files.mdd, "Program Version,{},{}{}", VerString, IDDVerString, DataStringGlobals::NL);
+        print(state.files.mdd, "Var Type (reported time step),Var Report Type,Variable Name [Units]{}", DataStringGlobals::NL);
```

Comment:
@mbadams5 @Myoldmopar This reintroduced `DataStringGloblas::NL` which was removed in #8178. The rdd and mdd files now have an extra line ending character in the header line `0D 0D 0A`. This causes text editors on Windows to prompt to fix line endings.